ingestion/functions/retrieval/retrieval_test.py

```diff
@@ -64,21 +84,20 @@ def test_lambda_handler_e2e(valid_event, requests_mock, s3):
     s3.create_bucket(Bucket=retrieval.OUTPUT_BUCKET)
 
     # Set up mock request values used in multiple requests.
-    source_api_url = "http://foo.bar"
-    os.environ["SOURCE_API_URL"] = source_api_url
+    os.environ["SOURCE_API_URL"] = _SOURCE_API_URL  # Still needed for now.
```

Comment:
for now? what do we need TODO to fix it? /wink

Reply:
Heh, thanks, done.